Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.cpp

```diff
@@ -104,17 +105,14 @@ int CLuaCameraDefs::GetCamera(lua_State* luaVM)
     return 1;
 }
 
-int CLuaCameraDefs::GetCameraViewMode(lua_State* luaVM)
+std::tuple<int, int> CLuaCameraDefs::GetCameraViewMode()
 {
-    unsigned short ucMode;
-    if (CStaticFunctionDefinitions::GetCameraViewMode(ucMode))
-    {
-        lua_pushnumber(luaVM, ucMode);
-        return 1;
-    }
+    CClientCamera* pCamera = g_pClientGame->GetManager()->GetCamera();
 
-    lua_pushboolean(luaVM, false);
-    return 1;
+    unsigned short ucVehicleMode = (unsigned short)pCamera->GetCameraVehicleViewMode();
+    unsigned short ucPedMode = (unsigned short)pCamera->GetCameraPedViewMode();
+
```

Comment:
I don't think those casts are needed. They're implicitly converted.
Also, it's `us` not `uc`

Reply:
It should be.